centos-5.11-i386.json

```diff
@@ -102,21 +102,18 @@
   ],
   "post-processors": [
     {
-      "output": "builds/{{.Provider}}/opscode_centos-5.11-i386_chef-{{user `chef_version`}}.box",
+      "output": "builds/{{.Provider}}/opscode_centos-5.11-i386.box",
```

Comment:
These kinds of strings are hardcoded here:

https://github.com/test-kitchen/kitchen-vagrant/blob/master/lib/kitchen/driver/vagrant.rb#L112

Needs to still be "_chef-provisionerless.box"